node/store.go

```diff
@@ -223,6 +223,13 @@ func (s *Store) deleteNBatches(currentTimeUnixSec int64, numBatches int) (int, e
 			size += int64(len(blobIter.Value()))
 		}
 		blobIter.Release()
+
+		// Blob index.
+		blobIndexIter := s.db.NewIterator(EncodeBlobIndexKeyPrefix(batchHeaderHash))
```

Comment:
If the intent is to mostly deprecate this function, should we put this code somewhere else?

Reply:
Which function are you referring to? `deleteNBatches` or `EncodeBlobIndexKeyPrefix`?

Reply:
Referring to `deleteNBatches`. It's currently handling both "Batches" which will be deprecated and "Batch Mappings" which won't be?

Reply:
Sure, agree having a separate method would make it easier to deprecate the functionality in the future